### PR TITLE
Add knn and query search field options to ElasticKnnSearch

### PR DESCRIPTION
**Pull request by jeffvestal**
in the `ElasticKnnSearch` class added 2 arguments that were not exposed properly

`knn_search` added:
- `vector_query_field: Optional[str] = 'vector'`
-- vector_query_field: Field name to use in knn search if not default 'vector'

`knn_hybrid_search` added:
- `vector_query_field: Optional[str] = 'vector'`
-- vector_query_field: Field name to use in knn search if not default 'vector'
- `query_field: Optional[str] = 'text'`
-- query_field: Field name to use in search if not default 'text'

Fixes # https://github.com/hwchase17/langchain/issues/5633

cc: @dev2049 @hwchase17

### CODE REVIEW

File: langchain/vectorstores/elastic_vector_search.py [hunk 1]
```diff
@@ -498,6 +500,8 @@ def knn_hybrid_search(
         fields: Optional[
             Union[List[Mapping[str, Any]], Tuple[Mapping[str, Any], ...], None]
         ] = None,
+        vector_query_field: Optional[str] = 'vector',
```
Comment (dev2049) [inline langchain/vectorstores/elastic_vector_search.py:503]: should both of these just be stored on the object?

Comment (jeffvestal) [reply langchain/vectorstores/elastic_vector_search.py:503]: Actually I'm with you now. I'll move them to the init and update this

Comment (jeffvestal) [reply langchain/vectorstores/elastic_vector_search.py]: @dev2049 I moved the two field names as optional args toin `__init__` 
Let me know if that was what you had in mind

**Review by hwchase17 — APPROVED** [review]
lgtm!

**Pull request merged**